http/http_rest_proxy.cc

```diff
@@ -330,6 +330,8 @@ perform(const std::string & verb,
 
         myRequest.add_option(CURLOPT_URL, uri);
 
+        myRequest.add_option(CURLOPT_MAXREDIRS, 20);
```

Comment:
I would suggest to keep this guy close to `CURLOPT_FOLLOWLOCATION` below, guarded by `if followRedirect`